wd_helper.au3

```diff
@@ -683,96 +682,58 @@ EndFunc   ;==>_WD_FrameLeave
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_FrameList
 ; Description ...: Retrieves a detailed list of the main document and all associated frames
-; Syntax ........: _WD_FrameList($sSession[, $bReturnAsArray = True[, $sFilter = ''[, $bReturnHTML = False]]])
+; Syntax ........: _WD_FrameList($sSession[, $bReturnAsArray = True])
 ; Parameters ....: $sSession            - Session ID from _WD_CreateSession
 ;                  $bReturnAsArray      - [optional] Return result as array? Default is True.
-;                  $sFilter             - [optional] RegExp pattern to filter frames with their HTML content. Default is ''.
-;                  $bReturnHTML         - [optional] Return frame document's HTML source? Default is False.
-; Return values .: Success - 2D array (with 6 cols) or string ( delimited with | and @CRLF )
+; Return values .: Success - 2D array (with 5 cols) or string ( delimeted with | and @CRLF )
```

Comment:
"delimited" is misspelled

Reply:
Thanks.
FIxed: [typo delimited](https://github.com/Danp2/au3WebDriver/pull/362/commits/66b70efcad58f9bd8847a9d925760a330ebed3ff)